tools/sdk-cli/src/main.rs

```diff
@@ -82,3 +80,40 @@ fn show_results(res: Result<String>) {
         Err(err) => error!("Error: {}", err),
     }
 }
+
+fn init_logging() {
+    let target = Box::new(
+        OpenOptions::new()
+            .create(true)
+            .append(true)
+            .open("cli.log")
```

Comment:
@ok300  Perhaps we create this file under the working dir passed from the user?

Reply:
Done